src/lib.rs

```diff
@@ -37,39 +37,81 @@ mod macos;
 use macos as platform;
 
 use std::env;
-use std::fs::File;
+use std::ffi::OsStr;
+use std::fs::{self, File};
 use std::io::BufReader;
 use std::io::{Error, ErrorKind};
 use std::path::{Path, PathBuf};
 
 use pki_types::CertificateDer;
 
+const ENV_CERT_FILE: &str = "SSL_CERT_FILE";
+const ENV_CERT_DIR: &str = "SSL_CERT_DIR";
```

Comment:
For new code I think our [style guide](https://github.com/rustls/rustls/blob/main/CONTRIBUTING.md) wants these module private `const`'s at the bottom of the file.

Reply:
Done

Reply:
(Feel free to hit "Resolve conversation" if you're reasonably sure you've addressed the issue.)

Reply:
I still see these `consts` at the top of the file. Maybe forgot a push?